### ISSUE TITLE
Issues with specification of coordinate reference systems

### ISSUE THREAD

**Issue by lewismc** [ESIPFed/science-on-schema.org#15]
Hi Folks,

Two issues...

I noticed recently that the DBPEDIA resource at [http://dbpedia.org/resource/Coordinate_reference_system](http://dbpedia.org/resource/Coordinate_reference_system) actually just redirects to [http://dbpedia.org/resource/Spatial_reference_system](http://dbpedia.org/resource/Spatial_reference_system). This begs the question... why don't we just specify the latter http://dbpedia.org/resource/Spatial_reference_system?

The second issue here is that I personally, think that the definition of a CRS or SRS is important enough for it to be feature in the Table of Contents. Would anyone object to that?

**Comment by lewismc** [comment]
Also, I would like to discuss entry of a WGS84 Ellipsoid type. This is somewhat linked to the following guidance on adding an `additionalProperty` for CRS definition, but it is NOT the same
```
{
  "@context": {
    "@vocab": "http://schema.org/",
    "datacite": "http://purl.org/spar/datacite/",
    "dbpedia": "http://dbpedia.org/resource/"
  },
  "@type": "Dataset",
  "name": "Removal of organic carbon by natural bacterioplankton communities as a function of pCO2 from laboratory experiments between 2012 and 2016",
  ...
  "spatialCoverage": {
    "@type": "Place",
    "geo": {
      "@type": "GeoShape",
      "line": "39.3280,120.1633 40.445,123.7878"
    },
    "additionalProperty": {
      "@type": ["PropertyValue", "dbpedia:Coordinate_reference_system"],
      "@id": "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
    }
  }
}
```

Any guidance on how we would specify the [WGS84](http://dbpedia.org/resource/World_Geodetic_System) type of reference coordinate system?

**Comment by ashepherd** [comment]
@lewismc I think the [W3C Basic Geo vocabulary](https://www.w3.org/2003/01/geo/) was written for WGS84 at: 

```geo: PREFIX <http://www.w3.org/2003/01/geo/wgs84_pos#>```

Do you think this property (CRS/SRS) is important enough that we should specify a property finstead of using the `schema:additionalProperty` strategy? Do you, or anyone else, know of a vocabularythat does a good job of this? If not, we could create our own property for the CRS/SRS.

@fils
@charlesvardeman

**Comment by lewismc** [comment]
@ashepherd 

> geo: PREFIX <http://www.w3.org/2003/01/geo/wgs84_pos#>

BOOM yes that's correct. 

> Do you think this property (CRS/SRS) is important enough that we should specify a property finstead of ...

No not really. I think using `schema:additionalPropeerty` is just fine. I do however think that this should be added to the table of contents otherwise is can be easily overlooked. I also think providing some guidance on availability of relevant IRI's would also be beneficial here. For example, at PO.DAAC we are generating Dataset.jsonld documents dynamically based upon our canonical database. The JavaScript we use to do this holds a mapping of our Ellipsoid-type e.g. `WGS84` to `http://www.w3.org/2003/01/geo/wgs84_pos#` which is what should be used as the `schema:additionalProperty/@id`

Also, what do you think about addressing the http://dbpedia.org/resource/Coordinate_reference_system actually redirect to http://dbpedia.org/resource/Spatial_reference_system?

**Comment by ashepherd** [comment]
agreed, 

I can create a table for these relevant IRIs and also change the docs to use `http://dbpedia.org/resource/Spatial_reference_system` instead of `http://dbpedia.org/resource/Coordinate_reference_system`.

**Comment by ashepherd** [comment]
Fixed in 5c65198adcf8ad6c86a37b74a867a818722d9b2c

**Comment by lewismc** [comment]
Thanks
